src/main/java/org/folio/inventory/resources/UpdateOwnershipApi.java

```diff
@@ -125,7 +137,116 @@ private void processUpdateHoldingsOwnership(RoutingContext routingContext) {
   }
 
   private void processUpdateItemsOwnership(RoutingContext routingContext) {
-    // should be implemented in MODINV-955
+    try {
+      final var context = new WebContext(routingContext);
+      final var updateOwnershipRequest = routingContext.body().asJsonObject();
+
+      final var validationError = updateOwnershipHasRequiredFields(context.getTenantId(), updateOwnershipRequest, ItemsUpdateOwnership.class);
+
+      if (validationError.isPresent()) {
+        unprocessableEntity(routingContext.response(), validationError.get());
+        return;
+      }
+      var itemsUpdateOwnership = updateOwnershipRequest.mapTo(ItemsUpdateOwnership.class);
+
+      List<NotUpdatedEntity> notUpdatedEntities = new ArrayList<>();
+
+      LOGGER.info("updateHoldingsOwnership:: Started updating ownership of item record: {}, to tenant: {}", itemsUpdateOwnership.getItemIds(),
```

Comment:
"updateHoldingsOwnership" - looks like log message should be corrected, and maybe LOG_UPDATE_ITEMS_OWNERSHIP constant used as in other places

Reply:
Done, Thanks